src/test/java/com/amazon/pvar/tspoc/merlin/InterproceduralPointsToTests.java

```diff
@@ -588,4 +588,26 @@ public void closureDepth3() {
         assert pts.contains(new ObjectAllocation(((NewObjectNode) getNodeByIndex(36, flowGraph))));
         assert pts.size() == 3;
     }
+
+    @Test
```

Comment:
Should we mark this as `@Ignore` while it doesn't yet pass. That way, we know that we haven't broken anything with other changes by checking whether all non-ignored tests pass.